### PR TITLE
Add /faq tiering in info.js

### PR DESCRIPTION
**Pull request by Rhythms88**

### CODE REVIEW

File: chat-plugins/info.js [hunk 1]
```diff
@@ -2299,6 +2299,10 @@ var commands = exports.commands = {
 			matched = true;
 			buffer += "<a href=\"http://www.smogon.com/forums/threads/coil-explained.3508013\">What is COIL?</a><br />";
 		}
+		if (target === 'all' || target === 'tiering' || target === 'tiers' || target === 'tier') {
+			matched = true;
+			buffer += "a href=\"https://www.smogon.com/ingame/battle/tiering-faq\">Tiering FAQ</a>";
```
Comment (panpawn) [inline chat-plugins/info.js]: I think you mean ``<a href...``

Comment (Rhythms88) [inline chat-plugins/info.js]: oop

Comment (panpawn) [inline chat-plugins/info.js]: Also, at every other line like this it appears they have a ``<br />`` tag at the end.

**Comment by Rhythms88** [comment]
squashed

File: chat-plugins/info.js (hunk 1 above)
Comment (Rhythms88) [inline chat-plugins/info.js]: oopoop

**Comment by Rhythms88** [comment]
k squashed again

**Pull request merged**